cpu/sam21_common/Makefile.include

```diff
@@ -1,6 +1,8 @@
-# export the CPU family so we can differentiate between them in the code
-FAM = $(shell echo $(CPU_FAM) | tr 'a-z-' 'A-Z_')
-export CFLAGS += -DCPU_FAM_$(FAM)
+# Define the CPU family so we can differentiate between them in the code
+CFLAGS += -DCPU_FAM_$(shell echo $(CPU_FAM) | tr 'a-z-' 'A-Z_')
+
+# Define the SAM0 family so we can differentiate between them in the code
+CFLAGS += -DSAM0_FAM_$(shell echo $(SAM0_FAM) | tr 'a-z-' 'A-Z_')
```

Comment:
Why do you pick on me? I'd say, look at your own contribution of cpu/saml21. It has the whole set of cmsis for saml21.

> IMHO we should just select the right cmsis header directly (in cpu_conf.h).

You can't. Not without adding a define like, eh, `SAM0_FAM_xyz`. If you had let me move samd21 to samr21 and add the 'real' samd21, then the three `samd21`, `samr21`, `saml21` would all have had the same structure. And `SAM0_FAM` wouldn't be needed.